### DISCUSSION TITLE
WS03 - Settings

### DISCUSSION THREAD

**Discussion by DanielSCBO** [arendst/Tasmota#11904] [Q&A] [answered]
Hello i I bought the WS03 (https://tasmota.github.io/docs/devices/WS03-WS01-WS05/)

But I don't know how to configure!

Does anyone know or have the template for this model?

Thanks

**Comment by barbudor** [comment, marked as answer]
There is a template for the [WS05](https://templates.blakadder.com/WS05.html)
Should be very close to what you are looking for

**Comment by DanielSCBO** [comment]
thanks its works, i change template for WS03

`{"NAME":"WS03","GPIO":[0,0,0,0,0,0,0,0,418,417,416,0,0,0],"FLAG":0,"BASE":18}`
